docs/breakfast/pancakes.md

```diff
@@ -1 +1,7 @@
 # Ingredients
+
+1 banana  
+1 egg  
+2 spoons oat flakes  
+cinnamon   
```

Comment:
cinnamon is very nice but it is not quantified

Reply:
At taste